src/test/runner.ts

```diff
@@ -254,6 +255,23 @@ function filterOnly(suite: Suite) {
   return false;
 }
 
+function filterTestFileByLine(suite: Suite, focusedTestFileLines: FilePatternFilter[]) {
+  const testFileLineMatches = (specFileName: string, specLine: number) => focusedTestFileLines.some(({re, line}) => {
+    re.lastIndex = 0;
+    return re.test(specFileName) && line === specLine
```

Comment:
What happened to the "missing semicolons" rule? :)

Reply:
Was busted on ToT, got fixed in #7268.